Firestore/Source/Core/FSTFirestoreClient.mm

```diff
@@ -121,6 +121,7 @@ @implementation FSTFirestoreClient {
   std::shared_ptr<AsyncQueue> _workerQueue;
 
   std::unique_ptr<RemoteStore> _remoteStore;
+  std::unique_ptr<EventManager> _eventManager;
```

Comment:
Can you use [`DelayedConstructor<EventManager>`](https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/Source/API/FIRListenerRegistration.mm#L28) instead?

@wilhuff Gil, I presume this is an appropriate usage of `DelayedConstructor`, right?

Reply:
Done.